config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml

```diff
@@ -1069,3 +1069,64 @@ periodics:
       - --test_args=--ginkgo.focus=\[sig-network\] --ginkgo.skip=\[Slow\]|\[Flaky\]|\[Feature:Networking-IPv6\]|\[Disruptive\]|\[Feature:ServiceLoadBalancer\]|\[Feature:PerformanceDNS\] --minStartupPods=8
       - --timeout=120m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20180730-8b7ab3104-master
+
+- interval: 2h
+  name: ci-kubernetes-e2e-containerd-gce-calico-serial
```

Comment:
It isn't obvious to me what this is testing, but it doesn't seem like the kubernetes/kubernetes repo given that this is using a fixed version of kubernetes (`v1.11.2-beta.0.20+21777efd99bd59`)

Reply:
it's k8s 1.11 with containerd, calico and netd enabled.